mysql-test/suite/innodb/r/foreign_null,INPLACE.rdiff

```diff
@@ -0,0 +1,27 @@
+--- foreign_null.result
++++ foreign_null,INPLACE.result
+@@ -48,7 +48,7 @@
+ FOREIGN KEY(f1) REFERENCES `t#1`(f2)
+ ON UPDATE CASCADE)ENGINE=InnoDB;
+ ALTER TABLE `t#1` MODIFY COLUMN f2 INT;
+-ERROR HY000: Cannot change column 'f2': used in a foreign key constraint 't#2_ibfk_1' of table 'test.t#2'
++ERROR HY000: Cannot change column 'f2': used in a foreign key constraint 't#2_ibfk_1' of table 'test.t@00232'
+ DROP TABLE `t#2`, `t#1`;
```

Comment:
This difference looks unacceptable. One of the messages is incorrectly using a `my_charset_filename` encoded name of the table.